documentation/README.md

```diff
@@ -70,4 +70,9 @@ Endpoint                                       | Functionality
 `GET /api/v3/posts`                            | [Get posts](endpoints/posts.md#retrieve-all-posts)
 `GET /api/v3/posts/:post_id`                   | [Get a post](endpoints/signups.md#retrieve-a-specific-post)
 `DELETE /api/v3/posts/:post_id`                | [Delete a post](endpoints/signups.md#delete-a-post)
-`PATCH /api/v3/posts/:post_id`                 | [Update a post](endpoints/signups.md#update-a-post)   
+`PATCH /api/v3/posts/:post_id`                 | [Update a post](endpoints/signups.md#update-a-post)
```

Comment:
Should this use `endpoints/posts` instead of `endpoints/signups`?

Reply:
ah yes great catch!